README.md

```diff
@@ -0,0 +1,6 @@
+# Winner takes all
+
+The objective of this application is to....
+
+# Contributors:
```

Comment:
Need to add remaining team members

Reply:
But I do not know their names. Probably you can do that later.